doc/classes/AudioEffectDelay.xml

```diff
@@ -194,30 +197,43 @@
 	</methods>
 	<members>
 		<member name="dry" type="float" setter="set_dry" getter="get_dry">
+			Output percent of original sound. At 0, only delayed sounds are output. Value can range from 0 to 1. Default value: [code]1[/code].
 		</member>
 		<member name="feedback/active" type="bool" setter="set_feedback_active" getter="is_feedback_active">
+			If [code]true[/code] feedback is enabled. Default value: [code]false[/code].
 		</member>
 		<member name="feedback/delay_ms" type="float" setter="set_feedback_delay_ms" getter="get_feedback_delay_ms">
+			Feedback delay time in milliseconds. Default value: [code]340[/code].
 		</member>
 		<member name="feedback/level_db" type="float" setter="set_feedback_level_db" getter="get_feedback_level_db">
+			Sound level for [b]Tap1 delay[/b]. Default value: [code]-6 dB[/code].
```

Comment:
I'm still not sure why these are bolded.  If it's a reference to the member variable, it should be [code]tap1/delay_ms[/code].